package.json

```diff
@@ -36,8 +36,11 @@
     "css-loader": "^0.21.0",
     "express": "^4.13.3",
     "extract-text-webpack-plugin": "^0.9.0",
+    "node-sass": "^3.4.1",
+    "normalize.css": "^3.0.3",
```

Comment:
@javivelasco yes, its failing without normalize.css, I've just tried to add only node-sass & react-addons-css-transition-group to devDependencies:
```
ERROR in ./app/style.scss
Module build failed: ModuleNotFoundError: Module not found: Error: Cannot resolve module '~normalize.css' in /Users/vyorkin/github/react-toolbox-example/app
    at Compilation.<anonymous> (/Users/vyorkin/github/react-toolbox-example/node_modules/webpack/lib/Compilation.js:228:38)
```

I'm on npm v3.3.6, it doesn't install [peerDependencies](https://github.com/react-toolbox/react-toolbox/blob/master/package.json#L47) by default.